data_products.tex

```diff
@@ -376,15 +377,15 @@ \subsubsection{Survey Property Maps}
 Survey Property Maps \citep{10.71929/rubin/2570315} summarize how properties such as observing conditions or exposure time vary across the observed sky.
 Each map provides the spatial distribution of a specific quantity at a defined sky position for each band by aggregating information from the images used to make the \texttt{deep\_coadd}.
 Maps are initially created per-\gls{tract} and then combined to produce a final consolidated map.
-At each sky location, represented by a spatial pixel in the \gls{HEALPix} grid, values are derived using statistical operations, such as minimum, maximum, mean, weighted mean, or sum, depending on the property.
+At each sky location, represented by a spatial pixel in the \gls{HEALPix}\citep{2005ApJ...622..759G} grid, values are derived using statistical operations, such as minimum, maximum, mean, weighted mean, or sum, depending on the property.
```

Comment:
Moving the general HEALPix citation up here, since it's not directly applicable to `healsparse` below (which is a much more recent @erykoff invention).